### PR TITLE
bugfix:Fix file path error in the Dockerfile

### LINKED ISSUE #6818: File path error in the Dockerfile
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
image

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

Minimal yet complete reproducer code (or URL to code):

### Ⅵ. Environment:

- JDK version(e.g. `java -version`):
- Seata client/server version:  
- Database version:
- OS(e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by xingfudeshi**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
Fix file path error in the Dockerfile

### Ⅱ. Does this pull request fix one issue?

fixes #6818 

### Ⅳ. Describe how to verify it
image
image
image

### CODE REVIEW

File: distribution/Dockerfile [hunk 1]
```diff
@@ -34,11 +34,13 @@ WORKDIR /$BASE_DIR
 
 # ADD FORM distribution
 ADD bin/ /seata-server/bin
+ADD conf/ /seata-server/conf
 ADD ext/ /seata-server/ext
-ADD target/ /seata-server/target
 ADD lib/ /seata-server/lib
-ADD conf/ /seata-server/conf
+ADD script/ /seata-server/script
```
Comment (funky-eyes) [inline distribution/Dockerfile:40]: 这个加进来干嘛？
Why is this added?

Comment (xingfudeshi) [reply distribution/Dockerfile:40]: Some script files may be used in the future, such as database initialization, etc
image

**Review by funky-eyes — APPROVED** [review]
LGTM
